mleap-xgboost/src/main/scala/ml/combust/mleap/xgboost/bundle/ops/XGBoostClassificationOp.scala

```diff
@@ -0,0 +1,62 @@
+package ml.combust.mleap.xgboost.bundle.ops
+
+import java.io.ByteArrayInputStream
+import java.nio.file.Files
+
+import biz.k11i.xgboost.Predictor
+import ml.combust.bundle.BundleContext
+import ml.combust.bundle.dsl.{Model, Value}
+import ml.combust.bundle.op.OpModel
+import ml.combust.mleap.bundle.ops.MleapOp
+import ml.combust.mleap.runtime.MleapContext
+import ml.combust.mleap.xgboost.{XGBoostBinaryClassificationModel, XGBoostClassification, XGBoostClassificationModel, XGBoostMultinomialClassificationModel}
+import resource.managed
+
+/**
+  * Created by hollinwilkins on 9/16/17.
+  */
+class XGBoostClassificationOp extends MleapOp[XGBoostClassification, XGBoostClassificationModel] {
+  override val Model: OpModel[MleapContext, XGBoostClassificationModel] = new OpModel[MleapContext, XGBoostClassificationModel] {
+    override val klazz: Class[XGBoostClassificationModel] = classOf[XGBoostClassificationModel]
+
+    override def opName: String = "xgboost.classifier"
```

Comment:
as below, should probably include them in Bundle.BuiltinOps for consistency

Reply:
I don't want to include these in builtin ops because they aren't part of mleap-spark or mleap-spark extension. xgboost is definitely a 3rd party integration outside the scope of standard transformers.